tty/uart16550/uart16550.c

```diff
@@ -242,6 +244,31 @@ static void poolthr(void *arg)
 }
 
 
+static void _uart_mkDev(uint32_t port)
+{
+	oid_t dir;
+	char path[12];
+	unsigned int i;
+
+	while (lookup("/", NULL, &dir) < 0) {
```

Comment:
not sure You need `/`, `create_dev` internally waits for `devfs`, You should create devices there and then bind to root when available (see DTR project)

Reply:
Thanks, it works without waiting on `/`, removed `while` loop.
Yeah, in `phoenix-pilot-project` there is the same binding as in `DTR` project.